.gitvote.yml

```diff
@@ -16,3 +16,6 @@ profiles:
         - cncf-toc
       exclude_team_maintainers: true
     close_on_passing: true
+  announcements:

```

Comment:
The announcements section should be inside one of the profiles @riaankleinhans (`default` or `1wk` in this case). This allows you to enable this feature only in some profiles, or using different categories for announcements on each profile when needed 🙂

Reply:
Update, thank you for the review.